biopet/biopet.wdl

```diff
@@ -87,19 +85,14 @@ task ExtractAdaptersFastqc {
         Boolean? outputAsFasta
 
         Int memory = 4
-        Float memoryMultiplier = 2.5
+        Float memoryMultiplier = 5 # This is ridiculous...
```

Comment:
Once we move to slurm we need to check whether these multipliers are necessary.
I will make an issue on JIRA to remind us.